packages/bull-vault/src/LeverageBull.sol

```diff
@@ -105,7 +114,7 @@ contract LeverageBull {
      * @param _ethInCrab eth in crab strategy
      * @param _squeethInCrab oSQTH debt of crab strategy
      * @param _crabTotalSupply total supply of crab tokens
-     * @return ETH deposited as collateral in Euler and borrowed amount of USDC
+     * @return ETH deposited as collateral in Euler and borrowed amount of USDC, and total ETH deposited as collateral in Euler
```

Comment:
ETH deposited as collateral in Euler, and borrowed amount of USD, and total ETH deposited as collateral in Euler